### PR TITLE
Provide a way to override default_plugins_host for all plugins at one time

### PR DESCRIPTION
**Pull request by egouraud-claranet**
https://issues.jenkins-ci.org/browse/INFRA-1060

### CODE REVIEW

**Comment by jhoblitt** [comment]
Gaw - I will kick travis when the cert issue is sorted out.

File: manifests/init.pp [hunk 1]
```diff
@@ -269,44 +274,45 @@
...
+  $manage_datadirs      = $jenkins::params::manage_datadirs,
+  $localstatedir        = $::jenkins::params::localstatedir,
+  $executors            = undef,
+  $slaveagentport       = undef,
+  $manage_user          = $::jenkins::params::manage_user,
+  $user                 = $::jenkins::params::user,
+  $manage_group         = $::jenkins::params::manage_group,
+  $group                = $::jenkins::params::group,
+  $default_plugins      = $::jenkins::params::default_plugins,
+  $default_plugins_host = $::jenkins::params::default_plugins_host,
```
Comment (jhoblitt) [inline manifests/init.pp]: All params need to have basic type validation.

Comment (jhoblitt) [inline manifests/init.pp]: I would really like to see a simple test in https://github.com/jenkinsci/puppet-jenkins/blob/master/spec/defines/jenkins_plugin_spec.rb that demonstrates changing this parameter changes the plugin download url.

Comment (egouraud-claranet) [inline manifests/init.pp]: oh sorry,
i'm going to fix that

**Comment by jhoblitt** [comment]
👍 This looks good.  I'm going to merge it even though the acceptance tests are failing over the tls cert.  Lets switch to `updates.jenkins.io` in a separate PR.

**Comment by cam-itv** [comment]
Are we able to override the download for the swarm plugin? At the moment it's unable to be changed, and is still dependent on the main update server
https://github.com/jenkinsci/puppet-jenkins/blob/master/manifests/master.pp#L9-L11

**Comment by egouraud-claranet** [comment]
@cam-itv , i think the PR https://github.com/jenkinsci/puppet-jenkins/pull/728 do the job

**Pull request merged**